src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/reporting/util/Cas3ReportsTestHelper.kt

```diff
@@ -89,7 +89,7 @@ class Cas3BedUtilisationBedspaceReportData(
   override val town: String?,
   override val postCode: String,
   override val roomName: String,
-  override val bedspaceStartDate: LocalDate?,
+  override val bedspaceStartDate: Instant?,
```

Comment:
@muhammad-elabdulla just to let you know I think this should be Instant.  I've changed it in the upgrade branch but if you want to change it in main branch that would be good?

Reply:
I think currently we distinguish between date with time and date without time by using LocalDate and OffsetDateTime. In this case is date without time

Reply:
I'll look into this. Speaking to dan the backing column is a timestamp hence changing to Instant works. I'll review the datamodel and decide where to 'fix' this.

Reply:
i've had a look, i think we just need to update code in convertToCas3BedUtilisationBedspaceReportData to convert what will now be an Instant into a LocalDate (which is effectively what the function is currently doing, but converting an OffsetDateTime into a LocalDate

Reply:
I don't get what you mean exactly you want to convert OffsetDateTime to LocalDate?

Reply:
Basically we need to mostly revert what was done here, the only bit that will change is on the JPA Entity changing OffsetDataTime to Instant. On Cas3BedUtilisationBedspaceReportData it will still be LocalDate

Reply:
Yes agreed, it's all been done on the mainline (including using Instant instead of Timestamp as Hibernate5 also supports that)